src/wallet.js

```diff
@@ -973,13 +978,13 @@ const wallet = {
    * @inner
    */
   cleanLoadedData() {
-    storage.removeItem('wallet:accessData');
-    storage.removeItem('wallet:data');
-    storage.removeItem('wallet:address');
-    storage.removeItem('wallet:lastSharedIndex');
-    storage.removeItem('wallet:lastGeneratedIndex');
-    storage.removeItem('wallet:lastUsedIndex');
-    storage.removeItem('wallet:lastUsedAddress');
+    StorageProxy.getStorage().removeItem('wallet:accessData');
```

Comment:
I think we should create a `const storage = StorageProxy.getStorage()` to avoid duplicate code.

Reply:
Done